### PR TITLE
Minor additions and modifications to the Fedora and CentOS Periodic Jobs

### PR DESCRIPTION
**Pull request by shwetaap**
**What this PR does / why we need it**:
1. Updated the Fedora quay repo
2. Added an additional latest tag to the image uploadd to the final repo
3. Cleanup during the test run to make resources available

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```
Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

### CODE REVIEW

File: automation/test-linux.sh [hunk 1]
```diff
@@ -78,8 +82,7 @@ delete_vm(){
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process ${template_option} -n $namespace -o json NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc delete vm $vm_name -n $namespace
```
Comment (ksimon1) [inline automation/test-linux.sh:85]: Can you please replace `oc process` command in windows test too?

Comment (shwetaap) [reply automation/test-linux.sh:85]: Done

**Pull request merged**